crates/breez-sdk/core/src/sdk.rs

```diff
@@ -1218,6 +1232,14 @@ fn process_success_action(
     Ok(Some(SuccessActionProcessed::Aes { result }))
 }
 
+fn emit_optionaly_payment_complete(event_emitter: &EventEmitter, payment: Payment) {
```

Comment:
The fn name is a bit misleading, maybe something like `emit_final_payment_status`